ui/src/components/layout/AuthState.js

```diff
@@ -32,10 +32,10 @@ class AuthState extends React.Component {
                     variant="text"
                     color="secondary"
                     size="small"
-                    title={"Log out"}
+                    title={'Log out'}
                     style={ basicButton}
                     startIcon={<VerifiedUserIcon />}
-                    onClick={this.openLogout}> Logged in as '{ username }'</TopbarButton>
+                    onClick={this.openLogout}> { username }</TopbarButton>
```

Comment:
Is this intentional?

Reply:
hmm it must come from a   different branch.  

It is good to mention 'Logged in as'  taking more space than anything else. showing the username with the icon 'check' is already a strong sign of being logged in.

Reply:
but the ' i will have to changed to &squo;

Reply:
Ok i'll go with your change